rust/core-lib/src/tabs.rs

```diff
@@ -19,6 +19,8 @@
 //! This file is called 'tabs' for historical reasons, and should probably
 //! be renamed.
 
+#![allow(clippy::clone_on_copy)]
```

Comment:
what's the rationale for this? I'd prefer to fix whatever the problem was.

Reply:
Me getting annoyed with clippy. Fixed.